### ISSUE TITLE
Error building Sitecore 9.1.0 XP base for ltsc2019

### ISSUE THREAD

**Issue by marcus-n3rd** [Sitecore/docker-images#55]
I during my attempts to build images for Sitecore 9.1.0, I receive the following error:

```powershell
### Processing 'sitecore-xp-base:9.1.0-windowsservercore-ltsc2019'...
Sending build context to Docker daemon  308.5MB
Step 1/9 : ARG BUILD_IMAGE
Step 2/9 : ARG BASE_IMAGE
Step 3/9 : FROM $BUILD_IMAGE as builder
 ---> f2f1c7beb43c
Step 4/9 : SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 ---> Using cache
 ---> ce514b2f8b91
Step 5/9 : COPY . /install/
 ---> Using cache
 ---> 3e32f653d520
Step 6/9 : RUN $env:SIF_CONFIG = 'createcert.json';     $env:INSTALL_TEMP = 'C:\\install';     Expand-Archive -Path (Join-Path $env:INSTALL_TEMP '*.zip') -DestinationPath $env:INSTALL_TEMP;     Expand-Archive -Path (Join-Path $env:INSTALL_TEMP '*Configuration files*.zip') -DestinationPath $env:INSTALL_TEMP;     Rename-Item -Path (Resolve-Path (Join-Path $env:INSTALL_TEMP $env:SIF_CONFIG)).Path -NewName 'config.json';
 ---> Using cache
 ---> cfa3b6ad1b14
Step 7/9 : FROM $BASE_IMAGE
 ---> 1b286d3d7168
Step 8/9 : COPY --from=builder ["/install/config.json", "/install/"]
failed to copy files: mkdir \\?: The filename, directory name, or volume label syntax is incorrect.
Failed: docker image build --isolation 'hyperv' --build-arg BUILD_IMAGE=mcr.microsoft.com/windows/servercore:ltsc2019 --build-arg BASE_IMAGE=sitecore-base:9.1.0-windowsservercore-ltsc2019
--tag 'sitecore-xp-base:9.1.0-windowsservercore-ltsc2019' 'C:\path\to\Sitecore\docker-images\repo\images\9.1.0 rev. 001564\windowsservercore\sitecore-xp-base'
At C:\path\to\Sitecore\docker-images\repo\modules\SitecoreImageBuilder\SitecoreImageBuilder.psm1:244 char:74
+ ... here-Object { $_ } | ForEach-Object { throw "Failed: $buildCommand" }
+                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Failed: docker ...tecore-xp-base':String) [], RuntimeException
    + FullyQualifiedErrorId : Failed: docker image build --isolation 'hyperv' --build-arg BUILD_IMAGE=mcr.microsoft.com/windows/servercore:ltsc2019 --build-arg BASE_IMAGE=sitecore-base:9.
   1.0-windowsservercore-ltsc2019 --tag 'sitecore-xp-base:9.1.0-windowsservercore-ltsc2019' 'C:\path\to\Sitecore\docker-images\repo\images\9.1.0 rev. 001564\windowsservercore\sitecore-xp-bas
  e'
```

Everything looks to be correct and the file _does_ exist in the builder image, but the next stage always fails.

Thanks in advanced.

**Comment by pbering** [comment]
Hi, thanks for reaching out! This is a known issue in the new Docker Engine 19.03.1 that was released together with Docker Desktop 2.1.0.0: https://github.com/docker/for-win/issues/4349.

If you only need a few of the images you can manually apply the workaround that I wrote in above issue, or downgrade Docker Engine to the previous 18.09.0 version.

Currently I'm investigating how we best can mitigate this problem here. I think the solution will be to apply the workaround until a new Docker Desktop and engine is released.

**Comment by pbering** [comment]
Fixed!
